### DISCUSSION TITLE
Error 0x80370102 (UBUNTU)

### DISCUSSION THREAD

**Discussion by JLCQ303** [hackatomik/SISCOx256#19] [Preguntas Técnicas | Lab]
la mejor manera que encontré para solucionar este error es gracias a este video: https://www.youtube.com/watch?v=lzZqQ64ml4Y&t

Si esto no funciona prueba entrando en la BIOS de tu ordenador y activando al virtualización.
Esto se puede ver entrando al administrador de tareas en la parte de rendimiento

**Comment by quantumquirkxyz** [comment]
Hola, gracias por compartir este datazo!!
